src/components/Sensitivity/Sensitivity.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Chip, makeStyles, Typography } from '@material-ui/core';
+import { VerifiedUserOutlined } from '@material-ui/icons';
+import { FC } from 'react';
+import * as React from 'react';
+import { Sensitivity as SensitivityType } from '../../api';
```

Comment:
Hmm this is probably bad DX since we would be constantly fighting which import - the component or the type.
Since the type name is auto-generated, it would be better if we could come up with a different name for the component. Any ideas?

Reply:
Agreed - thinking on it!

Reply:
`SensitivityStatus`? 🤷

Reply:
Let's punt lol